api/build.gradle

```diff
@@ -435,7 +466,6 @@ dependencies {
     // 5.4+ results in a runtime error:
     // java.lang.NoSuchMethodError: 'void org.apache.hc.core5.http.impl.io.DefaultHttpRequestWriterFactory.<init>(org.apache.hc.core5.http.config.Http1Config)'
     implementation "org.apache.httpcomponents.client5:httpclient5:5.3.1"
-    implementation "org.springframework.boot:spring-boot-starter-validation:$project.ext.SPRING_BOOT_VERSION"
```

Comment:
For most of the Spring dependencies:
* moved them up in the file
* grouped with other Spring packages
* removed version numbers
* removed exclusions (no longer needed)